<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleDS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="MainTab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="FRCLogo">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../resources/resources.qrc">:/pixmaps/frc-logo.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Robot Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="statusLabel">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Status">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>--.--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="voltageLabel">
             <property name="text">
              <string>Voltage</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="Voltage">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>--.--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ModesBox">
          <property name="title">
           <string>Mode</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="Test">
             <property name="text">
              <string>Test</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">ModesGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Autonomous">
             <property name="text">
              <string>Autonomous</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">ModesGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Teleoperated">
             <property name="text">
              <string>Teleoperated</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">ModesGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SettingsBox">
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="ProtocolLAbel">
             <property name="text">
              <string>Protocol</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="Protocols"/>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="TeamNumber">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="TeamStationLabel">
             <property name="text">
              <string>Team Station</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="TeamStation"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="TeamLabel">
             <property name="text">
              <string>Team number</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="Spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="RobotIPLabel">
             <property name="text">
              <string>Robot IP Address</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="RobotIP"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="JoysticksTab">
       <attribute name="title">
        <string>Joysticks</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="Joystick">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>208</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/resources.qrc">:/pixmaps/joystick.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="Table">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>450</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>450</number>
          </attribute>
          <row>
           <property name="text">
            <string>Left Stick</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right Stick</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>A, B, X, Y</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Left Trigger</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right Trigger</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Left Bumper</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right Bumper</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>D-Pad</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Keyboard Assignment</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>W,A,S,D</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>I,J,K,L</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>0,1,2,3</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>Q &amp; E</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>U &amp; O</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string>Arrows</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NetConsoleTab">
       <attribute name="title">
        <string>NetConsole</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="MessagesLabel">
          <property name="text">
           <string>Messages:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="Console">
          <property name="font">
           <font>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="WindowSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>32</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="Buttons" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Disable">
         <property name="text">
          <string>Disable</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">StatusGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Enable">
         <property name="text">
          <string>Enable</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">StatusGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="StatusGroup"/>
  <buttongroup name="ModesGroup"/>
 </buttongroups>
</ui>
